packages/cf-style-container/test/__snapshots__/index.js.snap

```diff
@@ -27,10 +27,6 @@ Object {
     "boxShadow": "0 0 20px 0 rgba(136,136,136,0.50)",
```

Comment:
Not sure why this snapshot update but I guess the snapshot wasn't updated. Any ideas?